### DISCUSSION TITLE
配置文件错误

### DISCUSSION THREAD

**Discussion by misakamikotoda** [mikumifa/biliTickerBuy#124] [提问专区]
如图显示配置文件错误，使用版本为v2.7.9（最新版打开后闪退不知道怎么解决
![EWYQ YMIL6`P@IYN1D5$S C](https://github.com/mikumifa/biliTickerBuy/assets/172960606/ad659413-d74a-4159-b007-d01aff2b664f)
![EWYQ YMIL6`P@IYN1D5$S C](https://github.com/mikumifa/biliTickerBuy/assets/172960606/c4450910-73a8-4227-8f25-d7c18639ad37)
）

**Comment by mikumifa** [comment]
看一下说明书

**Comment by mikumifa** [comment]
配置文件不长这样

**Reply by misakamikotoda** [reply]
每次点击生成配置文件后会提示：当前票信息已更改，请点击“获取票信息”按钮重新获取

**Reply by misakamikotoda** [reply]
然后就陷入“获取票信息”“生成配置”重新获取获取票信息”“生成配置”的无尽循环

**Reply by mikumifa** [reply]
等一下在新版的v2.8.1，在v2.7.9里面填的事url不是票id

**Reply by mikumifa** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/99951454/ef1d69ca-2f3f-4778-88ed-ffdbe53b0ff3)
仔细读说明
